### PR TITLE
fix issue #4 by display a white background

### PR DESCRIPTION
**Pull request by yurimalheiros**
Hello, I added a rect shape to the svg for display only the rectangle in the correct size.

### CODE REVIEW

**Comment by Bekt** [comment]
Thanks @yurimalheiros, looks great. Minor styling feedback.

File: invatar/svg.py [hunk 1]
```diff
@@ -28,14 +28,17 @@ def make_svg(**options):
...
-         width="{size}px" height="{size}px"
-        style="background-color: {bg}">
-      <text y="50%" x="50%"
-          text-anchor="middle" dominant-baseline="central"
-          fill="{color}"
-          style="font-family: {font_family}; font-size: {font_size}px">
-        {text}
-      </text>
+         width="{size}px" height="{size}px">
+        <g>
```
Comment (Bekt) [inline invatar/svg.py]: Unpad everything from this line and below 2 spaces.

File: invatar/svg.py [hunk 2]
```diff
@@ -28,14 +28,17 @@ def make_svg(**options):
...
+            <rect x="0" y="0" width="{size}px" height="{size}px"
+                  fill="{bg}" ></rect>
+            <text y="50%" x="50%"
+                  text-anchor="middle" dominant-baseline="central"
+                  fill="{color}" style="font-family: {font_family};
```
Comment (Bekt) [inline invatar/svg.py]: Move `fill` right next to x/y so that all of `style` fits in one line.

**Comment by yurimalheiros** [comment]
I tried to fix the style, if it still no 100% ok, please let me know.

File: invatar/svg.py [hunk 3]
```diff
@@ -28,14 +28,17 @@ def make_svg(**options):
...
-        {text}
-      </text>
+         width="{size}px" height="{size}px">
+      <g>
+        <rect x="0" y="0" fill="{bg}" width="{size}px" height="{size}px">
+        </rect>
+        <text y="50%" x="50%" fill="{color}"
+              text-anchor="middle" dominant-baseline="central"
+              style="font-family: {font_family};
+              font-size: {font_size}px">
```
Comment (Bekt) [inline invatar/svg.py]: Can you move `font-size` to the line above:
`style="font-family: {font_family}; font-size: {font_size}px"`

Comment (yurimalheiros) [inline invatar/svg.py]: sure

**Comment by Bekt** [comment]
Thanks!

**Pull request merged**